network/dime_message.ksy

```diff
@@ -0,0 +1,74 @@
+meta:
+  id: dime_message
+  title: DIME (Direct Internet Message Encapsulation) Message
+  xref:
+    wikidata: Q1227457
+  license: CC0-1.0
+  bit-endian: be
+  endian: be
+doc: |
+  Direct Internet Message Encapsulation (DIME)
+  is an old Microsoft specification for sending and receiving
+  SOAP messages along with additional attachments,
+  like binary files, XML fragments, and even other
+  SOAP messages, using standard transport protocols like HTTP.
```

Comment:
IMHO it makes sense to include a command to get a sample file here:

```suggestion
  SOAP messages, using standard transport protocols like HTTP.
  
  Sample file: `curl -L
  https://github.com/kaitai-io/kaitai_struct_formats/files/5890499/scanner.dump.gz
  | gunzip -c > scanner.dump`
```

Reply:
Good idea! Thanks